src/types/twilio.ts

```diff
@@ -0,0 +1,13 @@
+// Based off Twilio documentation - https://www.twilio.com/docs/usage/webhooks/sms-webhooks
+export interface ITwilioSmsWebhookBody {
+  SmsSid: string
+  SmsStatus: string
+  MessageStatus: string
```

Comment:
should we use  a string union of all known status types here instead of the generic `string` type?  to make it clearer